src/software/guides/go-device-logging.md

```diff
@@ -123,3 +123,16 @@ Vehicles will not necessarily return all of the above information; the list is b
 Seat belt and odometer requests are proprietary on most passenger cars. It is Geotab's goal to support seat belt and odometer across all the major vehicle manufacturers. If you are unable to obtain seatbelt or odometer requests for your vehicle, please contact Geotab Support. You can query the percentage of seat belt, odometer and other engine based data is supported for different vehicle types via MyAdmin.
 
 Seat belt use is logged on status change: a value of 1 represents the seat belt unbuckled, while a value of 0 represents the seat belt buckled. Odometer is logged both on ignition on and ignition off and every hour.
+
+#### Engine Hours
+It is often important in fleet tracking to look at a vehicle's engine hours. While Geotab strives to support this data on as many vehicles as possible, it is sometimes not possible. This means it is important to distinguish between our two engine hours types of StatusData in order to utilize them appropriately.
```

Comment:
Could you change this "This means it is important to distinguish between our two engine hours types of StatusData in order to utilize them appropriately." 
to
"As a workaround, Geotab provides two types of engine hours StatusData. It is important to distinguish between the two in order to utilize them appropriately."

Reply:
@RichElliottGeoTab ". it" Needs to be capitalized.

Reply:
@Stephen-F All fixed, should be good for merge